db/schema.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE Genre (
+    id INT PRIMARY KEY,
+    name VARCHAR(255),
+);
+
+CREATE TABLE Author (
+    id INT PRIMARY KEY,
+    name VARCHAR(255),
+    publisher VARCHAR(255),
+);
+
+CREATE TABLE Book (
+    id INT PRIMARY KEY,
+    title VARCHAR(255),
+    publisher VARCHAR(255),
+    can_be_archived BIT,
+);
+
+CREATE TABLE Label (
+    id INT PRIMARY KEY,
+    name VARCHAR(255),
+    can_be_archived BIT,
+);
+
+CREATE TABLE Items (
+    genre_id INT,
+    item_id INT,
+    FOREIGN KEY (genre_id) REFERENCES Genre(id),
+    FOREIGN KEY (item_id) REFERENCES Book(id),
+);
+
```

Comment:
The items class should act as a parent table having associations with all classes including one to many relationships and child relationship